source/components/StreamTwitch.vue

```diff
@@ -17,6 +17,7 @@ export default {
   },
   data(){
     return {
+      host: location.host,
```

Comment:
`location.hostname` is better because it doesn't include port. Using `host` doesn't work in dev environment (i.e. `https://localhost:3000`) or when we deploy this to other hosts using different port number